### PR TITLE
Set a reference to the TokenType enum in the rust Token object

### PR DESCRIPTION
**Pull request by izeigerman**

### CODE REVIEW

File: sqlglotrs/src/lib.rs [hunk 1]
```diff
@@ -13,8 +13,10 @@ pub use self::tokenizer::Tokenizer;
 #[derive(Debug)]
 #[pyclass]
 pub struct Token {
-    #[pyo3(get)]
+    #[pyo3(get, name = "token_type_index")]
```
Comment (tobymao) [inline sqlglotrs/src/lib.rs:16]: so this is token type index?

File: sqlglotrs/src/lib.rs [hunk 2]
```diff
@@ -41,6 +43,7 @@ impl Token {
     ) -> Token {
         Python::with_gil(|py| Token {
             token_type,
+            token_type_py: PyNone::get(py).into(),
```
Comment (tobymao) [inline sqlglotrs/src/lib.rs:46]: just instantiating a python none value?

**Review by tobymao — APPROVED** [review]

File: sqlglotrs/src/lib.rs (hunk 2 above)
Comment (izeigerman) [reply sqlglotrs/src/lib.rs:46]: Exactly, yes

File: sqlglotrs/src/lib.rs (hunk 1 above)
Comment (izeigerman) [reply sqlglotrs/src/lib.rs:16]: it is, yes. But the rust code base doesn't need to worry about this detail. Do you suggest renaming the attribute?

Comment (tobymao) [reply sqlglotrs/src/lib.rs:16]: it’s ok

**Pull request merged**